templates/Pages/bof_isc23.php

```diff
@@ -0,0 +1,82 @@
+<?php $this->assign('title', 'Birds-of-a-Feather - ISC 23'); ?>
+
+<nav id="breadcrumb">
+    <p>YOU ARE HERE</p>
+
+    <?php
+    $this->Breadcrumbs->add(__('HOME'), ['controller' => 'submissions', 'action' => 'latest']);
+    $this->Breadcrumbs->add(__('BOF'), ['controller' => 'pages', 'action' => 'display', 'bofs']);
+    $this->Breadcrumbs->add(__("SC'22"), ['controller' => 'pages', 'action' => 'display', 'bof-sc22']);
```

Comment:
We also need to update here to point to this page

Reply:
done